authorization/models.py

```diff
@@ -55,3 +56,37 @@ def has_access(self, user: AnyUser, payload: Payload, permission: Permission, in
         Check that <payload> has <permission> access to <instance>.
         """
         return False
+
+
+class OAuth2BearerToken(models.Model):
```

Comment:
I _think_ the bearer token can be a signed JWT, so it wouldn't need to be saved to the database: https://oauthlib.readthedocs.io/en/latest/oauth2/tokens/bearer.html